pkg/client/fake/client.go

```diff
@@ -19,10 +19,12 @@ package fake
 import (
 	"context"
 	"encoding/json"
+	"errors"
 	"fmt"
 	"strconv"
 	"strings"
 
+	kerrors "k8s.io/apimachinery/pkg/api/errors"
```

Comment:
nit: everywhere I've seen this package renamed in c-r it's been to `apierrors`.

```suggestion
	apierrors "k8s.io/apimachinery/pkg/api/errors"
```